components/CheckStatusResponses/CheckStatusNoRecord.tsx

```diff
@@ -0,0 +1,47 @@
+import { FC } from 'react'
+import { useTranslation } from 'next-i18next'
+
+export const CheckStatusNoRecord: FC<{}> = () => {
+  const { t } = useTranslation(['status', 'common'])
+  return (
+    <>
+      <div data-testid="no-record" className="my-5">
+        {t('no-record.cannot-give-status.description')}
+        <ul className="ml-5 list-inside list-disc">
+          <li>{t('no-record.cannot-give-status.list.item-1')}</li>
+          <li>{t('no-record.cannot-give-status.list.item-2')}</li>
+        </ul>
+      </div>
+      <div className="my-5">
```

Comment:
`div` should be a `p` and the `ul` should be oustisde of the `p`

Reply:
I think this can be revisited after we see what it looks like with your PR since I don't think this looks very good right now